llvm/tools/llvm-objcopy/ObjcopyOpts.td

```diff
@@ -146,6 +146,12 @@ def extract_main_partition
     : Flag<["--"], "extract-main-partition">,
       HelpText<"Extract main partition from the input file">;
 
+defm set_max_section_offset
+    : Eq<"set-max-section-offset",
```

Comment:
Looking at this again, I think `set` can be dropped from the option name too.

Reply:
Hmm, I agree, thanks.